File: rxjava/src/main/java/com/asemlab/samples/rxjava/models/Name.java
package com.asemlab.samples.rxjava.models;

import androidx.annotation.Keep;
import com.google.gson.annotations.SerializedName;


@Keep
//@Serializable
public class Name {

        @SerializedName("common")
        public String common;

        @SerializedName("official")
        public String official;
}

File: ktor/src/main/java/com/asemlab/samples/ktor/models/Name.kt
package com.asemlab.samples.ktor.models


import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


@Keep
//@Serializable
data class Name(
    @SerializedName("common")
    val common: String? = null,
    @SerializedName("official")
    val official: String? = null
)